README.md

```diff
@@ -72,8 +72,9 @@ To set up a development environment for the CLI
 3. Run `make all` to install dependencies and build binaries and assets
 4. Start a `kind` cluster like so: `KIND_CLUSTER_NAME=<some name> ./tools/kind-with-registry.sh`
 5. Run `./bin/gitops install --config-repo=<repo url>`
-6. Start the in-cluster API replacement job (powered by [http://tilt.dev](tilt.dev)) with `make cluster-dev`
-7. make or make unit-tests to ensure everything built correctly.
+6. Run `make tls-files`
```

Comment:
since we are generating tls keys in memory are these new tools and generating localhost keys really necessary?

Reply:
Fair question. The certificates generated in memory are not valid, but the ones generated here are. I guess this is a matter of preference.

Reply:
Given that we are providing steps and tools to generate valid certs and having invalid auto-generated in-memory certs is a bit unusual, I have personally never seen any other project doing that, shouldn't we just remove the invalid auto-generate ones?

Since also we set up our dev environment to use the valid certs with tilt that auto-generate option will be mostly unused.